app/controllers/concerns/response.rb

```diff
@@ -0,0 +1,5 @@
+module Response
+  def json_response(object, status = :ok)
+    render json: object, status:
+  end
+end
```

Comment:
- [ ] Kindly fix the syntax error in this file.
![image](https://user-images.githubusercontent.com/81932478/206853657-d4a11089-5ff6-4717-b000-a9e4fb745008.png)

Reply:
Dear Munsa1,

I don't have that error in my code base. You might want to check your end. What we are testing for is the API and this is my response from postman.

![image](https://user-images.githubusercontent.com/46789910/206854562-b5edc59f-ecb9-4d1a-8731-35ea2e8e41cf.png)

![image](https://user-images.githubusercontent.com/46789910/206854558-d20c640f-8cb9-42a0-bb22-6d53c1e845b4.png)

Reply:
please pull the repo again.

Reply:
Fixed error everything works fine @Munsa1 @danielufeli